### PR TITLE
initial commit of v2 rev changes - remove treecover2000 from inference

### PR DESCRIPTION
**Pull request by orianac**
and fix landsat bugs

@tcchiao - eyes would be great! also am i reading it correctly that the linter is failing in the all_tiles.ipynb? when i run pre-commit locally it doesn't seem to fail there so maybe i'm reading it wrong.

### CODE REVIEW

File: carbonplan_trace/v1/inference.py [hunk 1]
```diff
@@ -182,6 +182,11 @@ def make_inference(input_data, model):
     gc.collect()
     print(f'predicting on {len(input_data)} records')
     t0 = time.time()
+    # this might not work for xgboost - joblib is something sklearn uses a lot
+    # but not something xgboost uses too much
+    # parallelization at this step was only done with random forest. at that time
+    # dask threading wasn't working and so we tried joblib and it worked
+    # we'll see about xgboost! TBD
```
Comment (tcchiao) [inline carbonplan_trace/v1/inference.py:189]: we probably don't need these comments anymore

File: carbonplan_trace/v1/landsat_preprocess.py [hunk 2]
```diff
@@ -415,23 +415,25 @@ def scene_seasonal_average(
...
     scene_stores = fs.ls(landsat_bucket.format(year, path, row))
-
+    scene_stores += fs.ls(landsat_bucket.format(year + 1, path, row))
+    # for australia months will be 12-2005 , 01-2006, 02-2006,
     datestamps = make_datestamps(months, year)
     for scene_store in scene_stores:
-        for datestamp in datestamps:
-            if datestamp in scene_store:
-                valid_files.append(scene_store)
+        if '_T1' in scene_store:
```
Comment (tcchiao) [inline carbonplan_trace/v1/landsat_preprocess.py:422]: double checking that we think `_T1` would be present in the file name of all valid scenes? curious what it stands for

File: carbonplan_trace/v1/inference.py (hunk 1 above)
Comment (orianac) [reply carbonplan_trace/v1/inference.py:189]: deleted

File: carbonplan_trace/v1/landsat_preprocess.py (hunk 2 above)
Comment (orianac) [reply carbonplan_trace/v1/landsat_preprocess.py:422]: good point- perhaps was a bit cavalier here- it excludes tier 2 scenes which are flagged as having lower data quality than tier 1. there don't appear to be too many of them (maybe ~5% of all scenes), but we can keep them anyway to increase data. i'll confirm w jon.

File: carbonplan_trace/v1/training_prep.py [hunk 3]
```diff
@@ -77,16 +77,25 @@ def prep_training_dataset(
                     )
                     del landsat_ds
                     # add in other datasets
-                    data = add_aster_worldclim(data, tiles, lat_lon_box=bounding_box).load()
+                    if len(data) == 1:
+                        data[0] = add_aster_worldclim(
+                            data[0], tiles[0], lat_lon_box=bounding_box[0]
+                        ).load()
+                    else:
```
[the remark this answers is not in the document]
Comment (orianac) [reply carbonplan_trace/v1/training_prep.py:84]: done!

**Pull request merged**